### PR TITLE
Home Task 2 complete

### PR DESCRIPTION
**Pull request by Goshozavr**

### CODE REVIEW

File: js_basics_HT/task1.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const name = 'Barack Obama'
+const age = 62
+const isBlack = true
+const headHair = null 
+const undefinedData =
```
Comment (sttaran) [inline js_basics_HT/task1.js]: when you use `const` you should assign value while declaring the variable. You should syntax error now

Comment (sttaran) [reply js_basics_HT/task1.js]: use `let` instead
```js
let undefinedData
```
Please notice that in this case you should not use `=` operator

File: js_basics_HT/task5_1.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+//Зробів два віришення на вибір, друге мені більше до вподоби)
+
+//const radius = 15
+
+//console.log(Math.PI*Math.pow(radius,2))
+
+const radius = 15
+const S = (Math.PI*Math.pow(radius,2))
```
Comment (sttaran) [inline js_basics_HT/task5_1.js]:
```js
const circleArea = Math.PI * Math.pow(radius,2)
```

Comment (sttaran) [reply js_basics_HT/task5_1.js]: remember about camelCase and meaningful names for variables

File: js_basics_HT/task5_2.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const lenght = 2
+const weight = 3
+const SR = (lenght*weight)
```
Comment (sttaran) [inline js_basics_HT/task5_2.js]:
```js
const reactangleArea = lenght*weight
```

Comment (sttaran) [reply js_basics_HT/task5_2.js]: remember about camelCase and meaningful names for variables

File: js_basics_HT/task5_2.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+const lenght = 2
```
Comment (sttaran) [inline js_basics_HT/task5_2.js]: length

File: js_basics_HT/task5_2.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+const weight = 3
```
Comment (sttaran) [inline js_basics_HT/task5_2.js]: width

File: js_basics_HT/task5_3.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+const height = 5
```
Comment (sttaran) [inline js_basics_HT/task5_3.js:1]: height

File: js_basics_HT/task5_3.js [hunk 7]
```diff
@@ -0,0 +1,5 @@
+const height = 5 
+const radius = 3
+const SC = (Math.PI*Math.pow(radius,2)*height)
```
Comment (sttaran) [inline js_basics_HT/task5_3.js]:
```js
const cylinderVolume = Math.PI * Math.pow(radius,2) * height
```

**Review by sttaran — CHANGES_REQUESTED** [review]

File: js_basics_HT/task1.js (hunk 1 above)
Comment (Goshozavr) [reply js_basics_HT/task1.js]: is it only about 'undefinedData'? 
btw - there were no errors

File: js_basics_HT/task5_1.js (hunk 2 above)
Comment (Goshozavr) [reply js_basics_HT/task5_1.js]: S - means square (I'll replace it with Area)

File: js_basics_HT/task5_2.js (hunk 3 above)
Comment (Goshozavr) [reply js_basics_HT/task5_2.js]: *rectangle
SR - means rectangle square, I'll replace it with rectangleArea and weight with width

File: js_basics_HT/task5_2.js (hunk 4 above)
Comment (Goshozavr) [reply js_basics_HT/task5_2.js]: missclick, I'll fix it

File: js_basics_HT/task5_2.js (hunk 5 above)
Comment (Goshozavr) [reply js_basics_HT/task5_2.js]: missclick, I'll fix it

File: js_basics_HT/task5_3.js (hunk 6 above)
Comment (Goshozavr) [reply js_basics_HT/task5_3.js:1]: ?

File: js_basics_HT/task5_3.js (hunk 7 above)
Comment (Goshozavr) [reply js_basics_HT/task5_3.js]: SC - square cylinder (my bad, I've made a mistake after previous tasks)
does () play any role in this case?

Comment (sttaran) [reply js_basics_HT/task5_3.js]: does () play any role in this case? - it does not, it's just unnecessary here

**Review by sttaran — APPROVED** [review]

**Pull request merged**